packages/tradershub/tailwind.config.js

```diff
@@ -75,6 +75,7 @@ module.exports = {
                         'less-prominent-text': '#6e6e6e',
                         'primary-background': '#0e0e0e',
                         'prominent-text': '#ffffff',
+                        'red-light-background': '#3c2020',
```

Comment:
rather than adding a new color under the tailwind config its better for you to make an:

```
red : {
     light: ...,
     main: ..,
     dark: ..,
}
```